web_external/Viewer/style.styl

```diff
@@ -14,6 +14,10 @@
       .video
         flex 1
 
+  .no-annotation-message
```

Comment:
Having this reduce the size of the viewer panel is... unfortunate. Please consider replacing this style with:

```
position absolute
top 3px
right 3px
z-index 1
background rgba(255,224,224,0.5)
padding 0 0.5em
border 1px solid rgba(160,80,80,0.5)
color #400
```